### DISCUSSION TITLE
Penggunaan API kirimwa.id

### DISCUSSION THREAD

**Discussion by CyberLee88** [kirimwa-id/forum#59] [Pamerkan]
Saya sedang develop system untuk ajang musyawarah nasional salah satu organisasi hobby.
Dengan adanya kebutuhan mengirim qr code ke Whatsapp para peserta dari sabang sampai merauke.
Tadi malam saya sdh coba system API kirim wa.id hasil nya memuaskan.Respon cukup cepat. di bawah 1 detik, message sdh sampe di WA tujuan.Implementasi API juga gampang, dan saya sangat terbantu oleh contoh2 yg diberikan.Sampai pagi ini Link dengan WA pengirim saya masih OK blm ada tanda2 putus.
MANTUL Gan !!!
![Screenshot_20211029-084606](https://user-images.githubusercontent.com/93358985/139359829-f71d2b4e-15f2-4a5d-85e8-af0d44ef4c3f.png)

**Comment by rioastamal** [comment]
Terima kasih @CyberLee88 👍  sudah memposting penggunaan API KirimWA.id, ditunggu update untuk penggunaan-penggunaan yang lain.
